### ISSUE TITLE
Assess needs for DCP1 metadata updates

### ISSUE THREAD

**Issue by lauraclarke** [ebi-ait/hca-ebi-wrangler-central#5]
# A plan for any metadata updates which need to be applied to DCP1 data after it has been submitted to DCP2

This goal of this ticket is to assemble a catalogue of updates which are needed for the data which is being transferred from DCP1 to DCP2

There is an existing set of tickets in the private wrangling repo https://github.com/HumanCellAtlas/hca-data-wrangling/projects/10

There were issues raised as part of the small 'r' release:
https://docs.google.com/spreadsheets/d/1coHOJNQOhsEffW0hOWCCE8UIk1gx-VT-TUoWWj-GQhU/edit#gid=28226770 

All the AUDR issues for DCP1 datasets here: https://docs.google.com/spreadsheets/d/1B52QjioJ7CzOLXFIcB-Q8mfjiFSH0jzbfdNs_nce3P4/edit#gid=1663653290

Any issues found after pushing all DCP1 datasets through the experimental graph validation to spot potential issues for the pipelines team https://github.com/ebi-ait/ingest-graph-validator

One outcome from this task is a list of the needed updates. This will be a list which is interesting to many people in the DCP team so it needs to be publicly accessible and as easy as possible to understand/track

## Acceptance criteria

- [ ] list of updates that are needed by dataset
- [ ] push all the DCP1 datasets through the graph validator
- [ ] feeding requirements for making updates to data in TDR once the initial transfer of data

**Comment by zperova** [comment]
make last criteria a separate ticket

**Comment by mshadbolt** [comment]
@zperova to find the version with the list of AUDR updates for all datasets

**Comment by zperova** [comment]
added to the body of the issue

**Comment by lauraclarke** [comment]
@zperova I would like to discuss coordination of when updates happen compared to pipeline triggers etc with Broad/UCSC,

How do I find a summary of updates that are needed?
How can I quantify how many require changes to how entities are linked together (graph changes)
How can I quantify how many might impact analyses in either how they are run or what their output looks like?

**Comment by mshadbolt** [comment]
The sheet here and linked above summarises all the changes we currently have detected, Enrique may uncover more: https://docs.google.com/spreadsheets/d/1B52QjioJ7CzOLXFIcB-Q8mfjiFSH0jzbfdNs_nce3P4/edit#gid=1663653290

The 'graph changes' would be ones that require reingestion, apart from those that required reingestion due to file metadata changes/ supp file linking changes

There is a section of columns that shows changes that 'affects other components' which could be relevant to your third question. Though I am not sure that pipelines have decided exactly what metadata they will put in their outputs. 

I found this document https://docs.google.com/document/d/1B7gesgwB6nlyKyOnxXQnogDzcWxlHegI8RXMyDhQlOc/edit# that summaries what metadata pipelines uses to trigger analysis.

**Comment by lauraclarke** [comment]
To record the questions I need to raise with other folks at Broad and UCSC about updates.

What needs to be true about the new TDR import system before we can start updates?
Should a snapshot be created before updates start?
When in relation to updates should pipeline runs for the MVP public data snapshot happen?
There are updates which require changes to links.json, how do we propagate these types of updates?

Please add any others you can think of

**Comment by lauraclarke** [comment]
@ESapenaVentura How far through the DCP1 datasets did you get with the graph validator?

@mshadbolt Is it reasonable to declare that we understand the updates are that are needed for all DCP1 datasets aside from any new issues to by the graph validator?

I have started a google doc about questions here https://docs.google.com/document/d/1py61KTWAczb2caZ6yuVGdwUCYp6rr4T4kfjLtLxQxhQ/edit#

**Comment by ESapenaVentura** [comment]
The datasets have been all tested, I'm going through the results and checking with the latest spreadsheets. I'm planning to finish it by tomorrow/thursday

**Comment by lauraclarke** [comment]
@ESapenaVentura What causes the box to be ticked on this ticket?
https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/14

**Comment by ESapenaVentura** [comment]
The boxes that are ticked are for the datasets that I have looked the validation and posted the results in the AUDR ticket for that dataset.

**Comment by mshadbolt** [comment]
work for this epic has largely been consumed into and superseded by #71
